spec/model/team_model.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Team, type: :model do
+    let(:new_team) do Team.new(id: 1, team: "123", manager: 2)
```

Comment:
Looks good overall! Just put Team.new etc on a new line & indented under let and it will be good to go